src/aws/osml/model_runner/common/ensemble_boxes_nms.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2024 Amazon.com, Inc. or its affiliates.
+
+"""
+Non-Maximum Suppression (NMS) and Soft-NMS implementation for bounding boxes.
+
+This module implements standard NMS, linear Soft-NMS, and Gaussian Soft-NMS
+for bounding boxes with normalized coordinates. It supports weighted scores
+and multiple labels.
+
+Original implementation inspired by:
+- https://github.com/ZFTurbo/Weighted-Boxes-Fusion
+
+Author: ZFTurbo (https://kaggle.com/zfturbo)
+Refactored for internal use in OSML.
+"""
+
+import logging
+from typing import Any, List, Optional, Tuple
+
+import numpy as np
+from ensemble_boxes.ensemble_boxes_nms import cpu_soft_nms_float
```

Comment:
looks like this is still importing from the ensemble_boxes library.  We'll need to copy at least 1 additional file into our baseline and change this import.

Reply:
Nope - I removed this =)